### PR TITLE
D8UN-315 set up basic page

### PR DESCRIPTION
**Pull request by ooneill547**

### CODE REVIEW

**Review by soda736 — APPROVED** [review]
LG

File: config/odscni/config/core.extension.yml [hunk 1]
```diff
@@ -86,6 +86,7 @@ module:
   toolbar: 0
   twig_tweak: 0
   unity_common: 0
+  unity_file_migrations: 0
```
Comment (DuttonMa) [inline config/odscni/config/core.extension.yml:89]: We shouldn't install any migration modules in config, as we don't want them to be installed on any Platform.sh sites, we just need to install them locally when we're working with migrations (see Uregni).

**Review by DuttonMa — CHANGES_REQUESTED** [review]
Just one module to uninstall, otherwise looks great

File: config/odscni/config/core.extension.yml (hunk 1 above)
Comment (ooneill547) [reply config/odscni/config/core.extension.yml]: This has now been removed

**Review by soda736 — APPROVED** [review]
Looks good

**Review by DuttonMa — APPROVED** [review]

**Pull request merged**